README.md

```diff
@@ -93,6 +93,23 @@ This gem also includes a dependency on `rubocop-rspec` that will be updated to
 the latest compatible version each time that the MAJOR.MINOR version of `rubocop` 
 is updated.
 
+## Change Process
+
+This configuration is meant to represent the general opinion of Salsify's Ruby community around best practices for
+writing readable Ruby code. As a result, changes to this configuration should go through a discussion phase in the
+#rubocop-changes Slack channel to ensure the broader Salsify Ruby community is on board with the change.
+
+### Updating Dependencies
```

Comment:
Perhaps this section header should be a bit more specific since this gem has other dependencies e.g. bundler, rake, rspec. Maybe "Updating Rubocop"?